File: app/src/main/java/starbright/com/projectegg/data/local/model/response/StepResponse.java
/**
 * Created by Andreas on 15/8/2018.
 */

package starbright.com.projectegg.data.local.model.response;

import com.google.gson.annotations.SerializedName;

import java.util.List;

public class StepResponse {

    @SerializedName("number")
    private String mNumber;

    @SerializedName("step")
    private String mStep;

    @SerializedName("ingredients")
    private List<IngredientResponse> mIngredients;

    public String getNumber() {
        return mNumber;
    }

    public String getStep() {
        return mStep;
    }

    public List<IngredientResponse> getIngredients() {
        return mIngredients;
    }
}

File: app/src/main/java/starbright/com/projectegg/data/model/response/StepResponse.kt
/*
 * Copyright (c) by Andreas (oentoro.andreas@gmail.com)
 * created at 25 - 7 - 2020.
 */

/**
 * Created by Andreas on 15/8/2018.
 */
package starbright.com.projectegg.data.model.response

import com.google.gson.annotations.SerializedName

data class StepResponse(
    @SerializedName("number")
    val number: Int,

    @SerializedName("step")
    val step: String,

    @SerializedName("ingredients")
    val ingredients: List<IngredientResponse>
)